Source/Dafny/Compiler-Csharp.cs

```diff
@@ -738,7 +738,7 @@ public CsharpCompiler(ErrorReporter reporter)
     protected override void DeclareSubsetType(SubsetTypeDecl sst, ConcreteSyntaxTree wr) {
       var cw = (ClassWriter)CreateClass(IdProtect(sst.EnclosingModuleDefinition.CompileName), IdName(sst), sst, wr);
       if (sst.WitnessKind == SubsetTypeDecl.WKind.Compiled) {
-        var sw = new ConcreteSyntaxTree(cw.InstanceMemberWriter.IndentLevel);
+        var sw = new ConcreteSyntaxTree(cw.InstanceMemberWriter.RelativeIndentLevel);
```

Comment:
Is the "relative" in `RelativeIndentLevel` meant to say it's in logical indentation units, rather than in character-count units?

Reply:
I see now. No need to rename.